src/clj/rems/db/test_data.clj

```diff
@@ -163,13 +163,13 @@
     (assert (:success result) {:command command :result result})
     (:id result)))
 
-(defn create-dynamic-workflow! [{:keys [actor organization title handlers]
-                                 :as command}]
+(defn create-workflow! [{:keys [actor organization title type handlers]
+                         :as command}]
   (let [result (workflow/create-workflow!
                 {:user-id (or actor "owner")
                  :organization (or organization "abc")
                  :title (or title "")
-                 :type :dynamic
+                 :type (or type :workflow/master)
```

Comment:
Is this default needed? I would prefer tests creating data without the master workflow. Perhaps master workflow type should not be used ever?

Reply:
Master workflow is the only workflow which is guaranteed to have all commands enabled. Thus it's a good default for testing, unless specifically testing the whitelist of a custom workflow.